WordPress/gutenberg
# [Mobile] UI for size image settings

## Description
This PR adds a mobile picker component, and a PickerCell that uses it.
The Picker can be used stand alone too.

This picker will show the native ActionSheet on iOS and our custom BottomSheet on Android, as requested by Thomas.

This PR manages just the UI, so the size image settings are not updated after the selection.

![android-select](https://user-images.githubusercontent.com/9772967/52404981-90a4f180-2aca-11e9-9073-6cc13cf6ff05.gif)
![size](https://user-images.githubusercontent.com/9772967/52404994-969ad280-2aca-11e9-81fd-c998b91d887f.gif)

**To test:**
- Check out the related [`gutenberg-mobile` branch ](https://github.com/wordpress-mobile/gutenberg-mobile/pull/555)and run the app.
- Add an image block and add an image to it.
- Open the image settings.
- Press on `Image Size`.
- Check that the corresponding bottom sheet appears with the size options, as shown in the.
- Repeat these steps for iOS and for Android.
